core-services/prow/02_config/red-hat-storage/ceph-csi/_prowconfig.yaml

```diff
@@ -15,15 +12,3 @@ tide:
     - needs-rebase
     repos:
     - red-hat-storage/ceph-csi
-  - includedBranches:
-    - devel
```

Comment:
this still needs to stay, otherwise the devel branch requires BZs for syncing from upstream